packages/jest-cli/src/check_if_interative.js

```diff
@@ -0,0 +1,3 @@
+import isCI from 'is-ci';
+
+export default process.stdout.isTTY && !isCI;
```

Comment:
NIT: I think we could name this file `is_interactive` so that the import looks like 'import isInteractive from './is_interactive'...

I could even be nice if this was in something like `jest-util` and then we could use it in other places of the codebase like
```
packages/jest-cli/src/pre_run_message.js
packages/jest-cli/src/reporters/__tests__/default_reporter.test.js
packages/jest-cli/src/reporters/coverage_reporter.js
packages/jest-cli/src/reporters/default_reporter.js
packages/jest-cli/src/watch.js
packages/jest-util/src/clear_line.js
```